proto/subscription/subscription.proto

```diff
@@ -6,11 +6,14 @@ option go_package = "github.com/lavanet/lava/x/subscription/types";
 message Subscription {
   string creator = 1; // creator pays for the subscription
   string consumer = 2; // consumer uses the subscription
-  uint64 block = 3;
-  string plan_index = 4;
-  uint64 plan_block = 5;
-  uint64 duration = 6; // duration in months
-  uint64 expiry_time = 7;
-  uint64 usedCU = 8;
-  uint64 remainingCU = 9;
+  uint64 block = 3; // when the subscription was created
+  string plan_index = 4; // index (name) of plan
+  uint64 plan_block = 5; // when the plan was created
+  uint64 duration_total = 6; // total requested duration in months
```

Comment:
why do we need the total? looks like it is only used in tests

Reply:
so if someone will buy 4 month you will auto buy for him another 4 month? seems odd. auto-renewel should add just 1 month i think, but it is the products decision

Reply:
Not me personally :)
But, consider there is mechanism to auto-renew (on chain, or 3rd party service, doesn't matter).
Suppose I bought 1 month subscription - I'd like it to auto-renew by 1 month at a time going forward.
Suppose I bought yearly subscription, - I'd like it to auto-renew by 1 year at a time going forward (or I will lose the discount).